dpctl/tensor/_dlpack.pyx

```diff
@@ -291,14 +296,29 @@ cpdef to_dlpack_capsule(usm_ndarray usm_ary):
         for i in range(nd):
             shape_strides_ptr[i] = shape_ptr[i]
         strides_ptr = usm_ary.get_strides()
+        flags = usm_ary.flags_
```

Comment:
We can define this above with other Cython variables
```
cdef int flags = 0
```
which is how it was before the `flags` check was removed